source/MaterialXRuntime/RtNodeGraph.h

```diff
@@ -61,6 +61,12 @@ class RtNodeGraph : public RtNode
     /// Return an iterator over the nodes in the graph.
     RtPrimIterator getNodes() const;
 
+    /// Return any associated version string.	
+    const RtToken& getVersion() const;
+
+    /// Set the associated version string.	
+    void setVersion(const RtToken& value);
```

Comment:
Nickpick: Docs and variable name is different compared to getVersion/setVersion on RtNodeDef. For consistency you could change to the same convention here.